const noop = () => { }

const throwErr = err => {
  throw new Error(err)
}